snippets.go

```diff
@@ -57,6 +57,7 @@ type Snippet struct {
 		RawURL string `json:"raw_url"`
 	} `json:"files"`
 	RepositoryStorage string `json:"repository_storage"`
+	ProjectID         int    `json:"project_id"`
```

Comment:
Please follow the ordering used in the docs (https://github.com/xanzy/go-gitlab/blob/main/CONTRIBUTING.md).

Reply:
Thanks I've updated the order to match the docs.